WordPress/wordpress-playground
# Restore the importFile step that was accidentally removed in #780

The importFile step, which allows developers to import WordPress content bia WXR and WXZ files, got accidentally removed in Pull Request #780. This commit restores that step.

## Open questions

`importFile` sounds ambiguous. What if it was split into two distinct steps called `importWxr` and `importWxz`? Then, the new `importWpFiles` step could be called `importWpContentDirectory` instead.

## Follow-up work

The importFile step uses `DOMParser` which isn't available in Node.js. It could, instead:

* Call the appropriate import-related PHP functions directly, or...
* Use the HTML Tag Processor available in WordPress 6.3+ (and polyfill it in the older WordPress versions).

## Testing Instructions

Go to the following localhost URL and confirm the WordPress reflects the non-standard content it imported from WordPress theme unit tests data:

http://localhost:5400/website-server/#{%22steps%22:[{%22step%22:%22login%22},{%22step%22:%22importFile%22,%22file%22:{%22resource%22:%22url%22,%22url%22:%22https://raw.githubusercontent.com/WordPress/theme-test-data/master/themeunittestdata.wordpress.xml%22}}]}

In particular, the menu should say twitter.com, facebook.com etc

As the Playground codebase starts supporting using WordPress in unit tests, let's add a test case for this step.

Closes #833

CC @eliot-akira @aplamada
